app/renderer/components/main/main.js

```diff
@@ -402,12 +384,11 @@ class Main extends ImmutableComponent {
       windowActions.setDownloadsToolbarVisible(false)
     })
 
-    const self = this
     ipc.on(messages.BLOCKED_RESOURCE, (e, blockType, details) => {
-      const frameProps = frameStateUtil.getFrameByTabId(self.props.windowState, details.tabId)
-      frameProps && windowActions.setBlockedBy(frameProps, blockType, details.url)
+      windowActions.setBlockedBy(this.props.activeFrameKey, blockType, details.url)
     })
 
+    // TODO can we remove this?
     ipc.on(messages.BLOCKED_PAGE, (e, blockType, details) => {
```

Comment:
it was listening to a message but taking no action when fired iirc. I can't recall why I didn't remove but given it is doing nothing seems like a leftover so I'm fine removing.